sentry.ts

```diff
@@ -1,45 +0,0 @@
-import * as Sentry from "@sentry/node";
```

Comment:
Does the sentry sdk automatically monitor the jobs now?

Reply:
No. I decided to remove this file since we're not  using it.

We still need to handle stats syncer and blob propagation errors. 

I'll do it in a different PR

Reply:
Cool, i've opened an issue for that https://github.com/Blobscan/blobscan/issues/368